core/domain/user_services.py

```diff
@@ -2220,10 +2505,15 @@ def get_contributor_usernames(category, language_code=None):
     else:
         raise Exception('Invalid category: %s' % category)
 
-    return get_usernames(user_ids)
+    usernames = list(get_usernames(user_ids, strict=True))
```

Comment:
Why do you need to cast to a list?

Reply:
The return type of `get_usernames` is `Sequence`, so that's why I used the list here to convert the sequence to List. But now, instead of casting, the assertion is used to narrow down the type.